Gopkg.toml

```diff
@@ -96,3 +96,9 @@ required = ["github.com/golang/protobuf/jsonpb"]
 [[constraint]]
   branch = "master"
   name = "github.com/pingcap/tipb"
+
+[[constraint]]
+  name = "github.com/pkg/errors"
```

Comment:
why not using `pingcap/errors` directly?

Reply:
from @gregwebs 's view keep using pkg/errors make better compatiility https://github.com/pingcap/tidb/pull/7151#issuecomment-419429364

Reply:
got it.

Reply:
the good point is came when tidb need a third-part libray and it using pkg/errors too, use this way, all code use a same errors code.